atd-vze/src/queries/locations.js

```diff
@@ -14,21 +14,13 @@ export const GET_LOCATION = gql`
         collsn_desc
         count
       }
-    }
-    locationTotals: get_location_totals(
```

Comment:
[`get_location_totals`](https://github.com/cityofaustin/atd-vz-data/blob/94ce0ae83023170f1c30860f2ec20a74c7ed129a/atd-vzd/triggers/get_location_totals.sql) is a postgres function we expose through the Hasura API. It takes a date argument and returns crash counts for the given date range.

I've cross checked this a few times, and I can't quite tell what this is doing differently vs querying the `locations_with_crash_injury_counts` view directly. I'm pretty confident we can leave this behind, but I comfortable not knowing for sure until we pipe in an exact replica of production data for comparison.

Reply:
noting that i like that the 5 years logic is captured in the DB view now - especially since it hasn't changed all this time  👍

Reply:
this sounds good to leave behind! less cruft whoooo